samples/methods.go

```diff
@@ -0,0 +1,35 @@
+// Methods
+package main
+
+import (
+	"fmt"
+)
```

Comment:
Remove parens for consistency with other slides.

Reply:
Done.